### ISSUE TITLE
[Controller Request]: 

### ISSUE THREAD

**Issue by BatManok027** [ndeadly/MissionControl#611]
### Controller Name

Razer Raiju Mobile

### Current Controller Behaviour

Controller is not successful in pairing with the console

### Supported Features

- [X] Rumble/Vibration
- [ ] Motion Controls
- [ ] Player Number Indicator
- [ ] Battery Level Reporting

### Additional Details

[https://mysupport.razer.com/app/answers/detail/a_id/3590/~/razer-raiju-mobile-%7C-rz06-02800-support-%26-faqs](url)

**Comment by ndeadly** [comment]
Assuming Mission Control is installed and running correctly, controllers that won't connect to the console usually end up being Bluetooth LE controllers. Can you pair it with Windows and check the device properties under Device Manager? It should tell you there if it's an LE device. Not sure how this can be done on other OS's if you're not a Windows user though.

**Comment by BatManok027** [comment]
I will try to connect via wired and try to get the controller vendor and product id.

**Comment by BatManok027** [comment]
As of now i tried connecting via bluetooth to pc, but it wont be detected. But via android, bluetooth connection works just fine. But i found on other forum that they can use the controller wirelessly on pc but needs a driver. Maybe it is the problem. Anyway, thank you so much.

**Comment by ndeadly** [comment]
Usually a driver would just be for detecting the inputs correctly. The controller should still be able to connect at least as far as I know. If it's connecting under android you could use an app like nRF Connect. It's a tool for exploring Bluetooth LE devices. If it shows up in there while scanning you can be sure it's an LE controller.

**Comment by BatManok027** [comment]
![Screenshot_2023-04-22-00-24-43-706_no nordicsemi android mcp](https://user-images.githubusercontent.com/110001511/233687073-0721e406-f5a1-4f5c-b4ed-b3be726673f5.jpg)
This is what i've found during scanning. Controller not yet connected to android. Do you need anymore info? Should i connect it and send you the other info?

**Comment by ndeadly** [comment]
Yeah, it's an LE controller. That's all I need for now. It can't be supported until I've completed my work on Bluetooth LE support.

**Comment by BatManok027** [comment]
@ndeadly thank you so much for your extreme effort and being responsive!

**Comment by ndeadly** [comment]
@BatManok027 I'm experimenting with Bluetooth LE support now in the #testing channel of my discord server. Come join the party and see if your controller will connect.
